userinterface/planui.ui

```diff
@@ -112,17 +112,43 @@
        </widget>
       </item>
       <item>
-       <widget class="QPushButton" name="Reverse">
+       <widget class="QPushButton" name="exportCurrentRouteButton">
         <property name="text">
-         <string>Reverse Route</string>
+         <string>Export to File...</string>
         </property>
        </widget>
       </item>
       <item>
-       <widget class="QPushButton" name="exportCurrentRouteButton">
-        <property name="text">
-         <string>Export to File...</string>
+       <widget class="QGroupBox" name="groupBox_4">
+        <property name="title">
+         <string>Modify</string>
+        </property>
+        <property name="alignment">
+         <set>Qt::AlignCenter</set>
         </property>
+        <layout class="QVBoxLayout" name="verticalLayout_6">
+         <item>
+          <widget class="QPushButton" name="reverseButton">
+           <property name="text">
+            <string>Reverse</string>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QPushButton" name="splitButton">
+           <property name="text">
+            <string>Split</string>
```

Comment:
Split at...

... indicates that there will be a dialog, I would suggest to have append first and then split, because split will probably be used more infrequently